validation/td-validation.ttl

```diff
@@ -991,22 +990,18 @@
           message. 
           With the core vocabulary, it only includes
           content type information, but TD Context Extensions could
-          be applied. <span class="rfc2119-assertion" id=
-          "td-expectedResponse-default-contentType">If no
-          <code>response</code> name-value pair is provided, it
-          MUST be assumed
-          that the content type of the response is equal to the
-          content type assigned to the Form instance.</span> Note
-          that <code>contentType</code> within an
+          be applied.
+          <span class="rfc2119-assertion" id=
+          "td-expectedResponse-default-contentType">
+          If no <code>response</code> name-value pair is provided, it MUST be
```

Comment:
Not expected change but it makes sense and is more correct. This will make it annoying for actions with an input and output as we need to have contentType in form and in response objects. Some more thinking is needed as this will have implementation impact.